openaerostruct/docs/user_reference/mesh_surface_dict.rst

```diff
@@ -225,10 +225,10 @@ The surface dict will be provided to Groups, including ``Geometry``, ``AeroPoint
       - m
       - Control point of spar thickness distribution.
     * - skin_thickness_cp
-      - np.array([0.005, 0.025])
+      - np.array([0.005, 0.02])
       - m
       - Control point of skin thickness distribution.
-    * original_wingbox_airfoil_t_over_c
+    * - original_wingbox _airfoil_t_over_c
```

Comment:
Did you add a space here so it wraps to the next line in the table?

Reply:
Yes